Assets/HoloToolkit/Input/Scripts/Utilities/AttachToController.cs

```diff
@@ -11,20 +11,18 @@ namespace HoloToolkit.Unity.InputModule
     /// <summary>
     /// Waits for a controller to be instantiated, then attaches itself to a specified element
     /// </summary>
-    public class AttachToController : MonoBehaviour
+    public class AttachToController : ControllerFinder
     {
-#if  UNITY_WSA && UNITY_2017_2_OR_NEWER
-        public InteractionSourceHandedness Handedness { get { return handedness; } }
 
+#if UNITY_WSA && UNITY_2017_2_OR_NEWER
         [Header("AttachToController Elements")]
         [SerializeField]
-        protected InteractionSourceHandedness handedness = InteractionSourceHandedness.Left;
-
+        protected new InteractionSourceHandedness handedness = InteractionSourceHandedness.Left;
 #endif
-        public MotionControllerInfo.ControllerElementEnum Element { get { return element; } }
+        public new MotionControllerInfo.ControllerElementEnum Element { get { return element; } }
```

Comment:
Does this need to be `new`, or can it be removed and use the one found in `ControllerFinder`?

Reply:
Good catch. the one we need to override is the protected serialized field enum *element* line 25. Deleting this duplicate.